chucks-pick3-web/assets/css/style-sheet.css

```diff
@@ -32,7 +32,7 @@ p {
 }
 .logo-container {
     display: block;
-    background-color: blueviolet;
+    background-color: rgb(0, 0, 0);
```

Comment:
Any particular reason why we are using RGB instead of color hash code?  Just wondering if there are any advantages to doing one or the other.